src/cloudant/credentials.py

```diff
@@ -13,11 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-_credentials_
-
-Utilities to support using a ~/.cloudant INI style configuration file
-to allow users to pass credentials in
-
+Module providing utilities to support using an INI style configuration file
+to allow users to pass credentials in.
```

Comment:
Ending the sentence with 'in' sounds awkward.  I think it makes sense without it.